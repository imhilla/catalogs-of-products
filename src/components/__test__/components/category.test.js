import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import CategoryFilter from '../../categoryFilter';

it('renders without crushing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<CategoryFilter />, div);
});

describe('<CategoryFilter />', () => {
  describe('render()', () => {
    const component = shallow(<CategoryFilter />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
