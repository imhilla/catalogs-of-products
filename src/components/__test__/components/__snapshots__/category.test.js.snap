// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[' 1'] = `
<div>
  <div>
    <div>
      <select
        onChange={[Function]}
        value="All"
      >
        <option
          key="CATEGORIESone"
          value="CATEGORIES"
        >
          CATEGORIES
        </option>
        <option
          key="poisonone"
          value="poison"
        >
          poison
        </option>
        <option
          key="fireone"
          value="fire"
        >
          fire
        </option>
        <option
          key="flyingone"
          value="flying"
        >
          flying
        </option>
        <option
          key="waterone"
          value="water"
        >
          water
        </option>
        <option
          key="bugone"
          value="bug"
        >
          bug
        </option>
        <option
          key="normalone"
          value="normal"
        >
          normal
        </option>
        <option
          key="electricone"
          value="electric"
        >
          electric
        </option>
        <option
          key="groundone"
          value="ground"
        >
          ground
        </option>
        <option
          key="grassone"
          value="grass"
        >
          grass
        </option>
        <option
          key="psychicone"
          value="psychic"
        >
          psychic
        </option>
        <option
          key="rockone"
          value="rock"
        >
          rock
        </option>
        <option
          key="fightingone"
          value="fighting"
        >
          fighting
        </option>
      </select>
    </div>
  </div>
</div>
`;
